### PR TITLE
Newsletters: add 297 (2024-04-10)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic entries
- [x] Bitcoin PR Review Club @stickies-v 

@stickies-v here's the source to last month's review club section for reference: https://raw.githubusercontent.com/bitcoinops/bitcoinops.github.io/master/_posts/en/newsletters/2024-03-13-newsletter.md  Please don't hesitate to ask here, on IRC, or wherever if you have any questions.  Thanks for writing!  (And many thanks to @LarryRuane for his excellent work on this section every month since Aug 2022!)

### CODE REVIEW

**Comment by stickies-v** [comment]
I don't think I have write access here, so I've pushed a commit to add review club summary to my own repo: https://github.com/bitcoinops/bitcoinops.github.io/commit/1031b8cd7db76eb41e8948c4439a804929a7cf02

I tried to be mindful of the formatting and scripts used here but I'm not very familiar with this style so please let me know if something's off?

**Comment by LarryRuane** [comment]
@stickies-v - Looks pretty good, two small things: a0 needs a close quote, and we tend to write shorter text source lines (up to around 80 columns), just because it makes the diffs easier to read if there are later edits. I didn't review for content but will try to do that in the next day or so. Thanks for taking this on!

Very minor suggestion, this might sound funny, but I've found it useful to "find on page" the question mark character when looking at the review club page (with the logs) in the browser -- the questions stand out that way and I can see if I can incorporate any of them in my writeup. (There are often questions that come up during the discussion that weren't written by the host ahead of time.)

(I had the same write access problem, but Dave fixed it.)

Another tip, in case no one mentioned it yet, don't force-push (either your initial commit or later corrections), because that could back out someone else's commit since we have multiple people pushing to the same branch. (I did that at least once!) Just do normal pushes, and Mike always squashes just before merging.

File: _posts/en/newsletters/2024-04-10-newsletter.md
Comment (LarryRuane) [inline _posts/en/newsletters/2024-04-10-newsletter.md:240]:
```suggestion
  [indicating][corallo free money] that said they'll adopt LND's method---and
```

File: _posts/en/newsletters/2024-04-10-newsletter.md
Comment (bitschmidty) [inline _posts/en/newsletters/2024-04-10-newsletter.md:258]:
```suggestion
{% assign day_after_posting = page.date | date: "%s" | plus: 86400 | date: "%Y-%m-%d 16:00" %}
```

File: _posts/en/newsletters/2024-04-10-newsletter.md
Comment (bitschmidty) [inline _posts/en/newsletters/2024-04-10-newsletter.md:107]:
```suggestion
  [small incompatibility][core lightning #7174]" between Core Lightning's
```

File: _posts/en/newsletters/2024-04-10-newsletter.md
Comment (bitschmidty) [inline _posts/en/newsletters/2024-04-10-newsletter.md:140]:
```suggestion
  [descriptor][topic descriptors] for a user based on the settings they want and then add
```

File: _posts/en/newsletters/2024-04-10-newsletter.md
Comment (bitschmidty) [inline _posts/en/newsletters/2024-04-10-newsletter.md:269]: I think this is the wrong link

**Comment by stickies-v** [comment]
Thanks for the review, @LarryRuane and @bitschmidty . I've pushed an updated commit  8cb85fc104a9b55b46436dc58f8b7b9a186288b9 that:
- fixes the missing quote issue (verified by actually building the website this time)
- uses the proper linker mechanism for the BIP and bitcoin/bitcoin/pull
- wraps to line length 72

File: _posts/en/newsletters/2024-04-10-newsletter.md
Comment (harding) [reply _posts/en/newsletters/2024-04-10-newsletter.md:280]: Leaving as-is.  I think "said" doesn't make sense in this sentence that already contains "indicating".

File: _posts/en/newsletters/2024-04-10-newsletter.md
Comment (harding) [reply _posts/en/newsletters/2024-04-10-newsletter.md:309]: Ooh, good catch, thanks!

**Comment by harding** [comment]
Reviewed @stickies-v's section, looks great, thanks!  I made one small edit to pass our tests and one small edit to clarify a remark (since BIP62 was never activated, minimial-size pushes are only required in relay/mempool for legacy transactions, although I believe they're required for both segwit v0 and v1 at the consensus layer).

Made edits (or left a reply) for all review comments to date (thanks @LarryRuane @bitschmidty !), added lede, releases/RCs, and topic entries.  This should be ready to go, but I'll check back later to see if there are any additional reviews.

Thanks everyone!

File: _posts/en/newsletters/2024-04-10-newsletter.md
Comment (vostrnad) [inline _posts/en/newsletters/2024-04-10-newsletter.md:126]: Perhaps clearer:
```suggestion
  example, zero can be encoded as the empty vector,<!-- skip-test --> `0x00`, `0x0000`, ... [Bitcoin
```
Could also be avoided by choosing a different number.

**Review by LarryRuane — APPROVED** [review]
ACK e27ce2a7e912feef732bc5e32ee46c4d151672c8

File: _posts/en/newsletters/2024-04-10-newsletter.md
Comment (harding) [reply _posts/en/newsletters/2024-04-10-newsletter.md:126]: Changing to `OP_0` since that's a synonym for the empty byte array.

**Comment by harding** [comment]
Addressed @vostrnad additional comments (thanks!  and thanks @LarryRuane for re-reviewing!)

**Review by bitschmidty — APPROVED** [review]
ACK, fixups, lede, topics, PR Review Club

File: _posts/en/newsletters/2024-04-10-newsletter.md
Comment (carlaKC) [inline _posts/en/newsletters/2024-04-10-newsletter.md:200]: hey opetch! sorry to be this week's "um, actually" but https://github.com/lightningnetwork/lnd/pull/8160 adds forwarding support to LND so that it can be used by other nodes in the network for blinded path creation. It's just disabled because we need https://github.com/lightningnetwork/lnd/pull/8485 to be fully spec-compliant with error handling (NB for blinded path privacy), and it'll be turned on by default once that goes through.

Sending support (on some APIs) was added in https://github.com/lightningnetwork/lnd/pull/7267, though admittedly the PR title is terrible so it's not particularly clear.

**Pull request merged**
